src/util.h

```diff
@@ -41,8 +41,8 @@ class CTranslationInterface
     boost::signals2::signal<std::string (const char* psz)> Translate;
 };
 
-extern std::map<std::string, std::string> mapArgs;
-extern std::map<std::string, std::vector<std::string> > mapMultiArgs;
+extern std::map<std::string, std::vector<std::string> > _mapMultiArgs;
+static const std::map<std::string, std::vector<std::string> >& mapMultiArgs = _mapMultiArgs;
```

Comment:
I believe this instantiates a `mapMultiArgs` variable into every .cpp file (all initialized as the same reference).

This will work, I think:
```diff
util.cpp:
 map<string, vector<string> > _mapMultiArgs;
+static std::map<std::string, std::vector<std::string> > mapMultiArgs_;
+const std::map<std::string, std::vector<std::string> >& mapMultiArgs = mapMultiArgs;
 bool fDebug = false;

util.h:
-extern std::map<std::string, std::vector<std::string> > _mapMultiArgs;
-static const std::map<std::string, std::vector<std::string> >& mapMultiArgs = _mapMultiArgs;
+extern const std::map<std::string, std::vector<std::string> >& mapMultiArgs;
 extern bool fDebug;